File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/main/java/simulacionTaller/simulacionTaller.java
package simulacionTaller;

import simulacionTaller.factories.WorkShopFactory;
import simulacionTaller.models.PersonBrench;

import static simulacionTaller.utils.funcionesMenu.menuWorkShop;

public class simulacionTaller {
    public static void main(String[] args) throws InterruptedException {
        // Instanciamos y ejecutamos el menú
        PersonBrench[] workBench = WorkShopFactory.create(54, 10, 20, 20);
        menuWorkShop(workBench);

    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/main/kotlin/simulacionTaller/simulacionTaller.kt
package simulacionTaller

import simulacionTaller.factories.WorkShopFactory
import simulacionTaller.utils.menuWorkShop

fun main(args: Array<String>) {
    // Instanciamos y ejecutamos el menú
    val workBench = WorkShopFactory.create(50, 10, 20, 20)
    menuWorkShop(workBench)
}
